python/lsst/ctrl/oods/gen3ButlerIngester.py

```diff
@@ -273,10 +290,15 @@ async def clean(self):
         )
         t = t - td
 
+        LOGGER.info("createButler()")
```

Comment:
To be honest, it looks like the DEBUG level would be more appropriate level for message like this. If you would like keep it at the INFO level please make it a little bit more "human friendly", e.g. "Creating a Butler".

The same applies to the message in line 297.

Reply:
Fixed.   It's going to be left in as INFO for now because we're measuring the amount of time this takes.

Reply:
Consider using 

```python
with lsst.utils.timer.time_this(...):
    do_your_work
```
which will time it for you and put the timings in a log message automatically.